cadc-test-vos/src/main/java/org/opencadc/conformance/vos/FilesTest.java

```diff
@@ -271,6 +285,59 @@ public void emptyFileTest() throws Exception {
 
     }
 
+    @Test
+    public void testDataNodePermission() {
+        try {
+            // Create a DataNode.
+            String nodeName = "files-data-node-permissions";
+            String nodePath = nodeName;
+            URL nodeURL = getNodeURL(nodesServiceURL, nodePath);
+            VOSURI nodeURI = getVOSURI(nodePath);
+            DataNode testNode = new DataNode(nodeName);
+            testNode.owner = authSubject;
+            testNode.isPublic = false;
+            log.debug("testNode: " + nodeURL);
+
+            // Cleanup leftover nodes
+            delete(nodeURL, false);
+
+            // PUT the nodes
+            put(nodeURL, nodeURI, testNode);
+
+            // user without permissions should be able to read the node
```

Comment:
This is FilesTest so it is testing the /files endpoint; all you have here is re-testing some /nodes behaviour

line 307: user with permission on parent can read the node metadata; this is either covered by NodesTest already or really trivial test setup verification (basically, is /int-tests public?)

line 315: why are you testing updating node props in the FilesTest? irrelevant

where is: user with read permission on parent but not DataNode should not be able to read the bytes (403)?

where is: set DataNode.isPublic = true and repeat previous, now OK (200)?

Reply:
In a rush to finish before I had to leave, and just getting it done instead of paying attention to what I was doing. No excuse for that. Rewrote the test the proper way.